modules/sc-mesh-secure-deployment/src/server-mesh.py

```diff
@@ -87,12 +87,25 @@ def add_message(uuid):
         return 'Not Valid Certificate'
 
 
+def verify_certificate(old, new):
+    """
+    Here we are validating the hash of the certificate. This is giving us the integrity of the file, not if the
```

Comment:
Like the client-mesh.py - this should link to a github issue so we can PR it later.